angular-ngrx-scss/.env.example

```diff
@@ -0,0 +1,5 @@
+JWT_SECRET=7ZkCMuD69N2WBBDCDzxTTYvsNG8qPJHWeNYe37iZTkmQLUmujZUZ3RnkqP6yfxTU
```

Comment:
I hope this is not a valid jwt secret.

- [ ] make sure that the example env file does not contain a valid or otherwise usable jwt secret

Reply:
I'll clear it out - it's copied straight from the ang/apollo/tw project so it's there too 😬 lol but yeah should have taken that out anyways.